core/src/main/scala/HelmOp.scala

```diff
@@ -21,6 +21,14 @@ object ConsulOp {
 
   final case class HealthCheck(service: String) extends ConsulOp[String]
 
+  final case class AgentRegisterService(
+    service: String,
+    id:      Option[String],
+    tags:    Option[NonEmptyList[String]],
+    address: Option[String],
+    port:    Option[Int]
```

Comment:
I'm cool with adding the other service API calls. Specifically, it looks like that'd be:
- List Services
- Deregister Service
- Enable Maintenance Mode

Regarding adding the missing parameters to AgentRegisterService, sure. It looks like the only I haven't added here are the EnableTagOverride, Check, and Checks.

Welp, I'll work on that stuff now. Thanks!